backend/src/v5/models/users.js

```diff
@@ -165,4 +167,30 @@ User.ensureIndicesExist = async () => {
 	}
 };
 
+User.getMemberInfoFromId = async (userId) => {
```

Comment:
again if we need this function this should be done in the processor.

If this needs to stay here for whatever reason then the naming is wrong, `getMember` is a teamspace context, we're looking at user records in dependent of teamspaces here.

Reply:
Having read the other file I think what you really want here is just a function that given an array of userIds it returns you the array of corresponding usernames?